adafruit_simple_text_display.py

```diff
@@ -177,7 +183,7 @@ def __getitem__(self, item):
                 )
         return self._lines[item]
 
-    def add_text_line(self, color=(255, 255, 255)):
+    def add_text_line(self, color: Tuple[int, int, int] = (255, 255, 255)) -> label:
```

Comment:
This should return `label.Label`

Reply:
Also, goood catch on the input, looks like we should create an issue in `adafruit_display_text` to update the type annotation for `Label` to take `Tuple[int, int, int]` for `color` as well.